src/ch/ehi/ili2db/toxtf/ToXtfRecordConverter.java

```diff
@@ -353,7 +356,21 @@ public String addAttrToQueryStmt(StringBuffer ret, String sep, String tableAlias
 						 ret.append(geomConv.getSelectValueWrapperCoord(makeColumnRef(tableAlias,attrSqlName+DbNames.ITF_MAINTABLE_GEOTABLEREF_COL_SUFFIX)));
 					 }
 				 }
-			 }else if(type instanceof CoordType){
+			}else if(type instanceof MultiSurfaceOrAreaType){
```

Comment:
kann vereinfacht werden, da es in ITF kein MultiSurfaceOrAreaType gibt (also createItfLineTables und createItfAreaRef weglassen)